src/Angor/Client/Components/Icon.razor

```diff
@@ -14,7 +15,17 @@
 
 @if (!string.IsNullOrEmpty(svgContent))
 {
-    @((MarkupString)svgContent)
+    if (@Class != "")
+    {
+        <i class="@Class">
+            @((MarkupString)svgContent)
+        </i>
+    }
+    else
+    {
+        @((MarkupString)svgContent)
```

Comment:
svgContent is always provided by our code, and not by users or query strings? So no injection possible here?